sdk/pixDomain/pixDomain.js

```diff
@@ -0,0 +1,43 @@
+const rest = require('../utils/rest.js');
+const SubResource = require("../utils/subResource").SubResource;
+
+
+class PixDomain extends SubResource {
+    /**
+     *
+     * PixDomain object
+     *
+     * @description The PixDomain object displays the QR Code domain certificate information of Pix participants.
```

Comment:
Sugestão:
`The PixDomain object displays the domain name and the QR Code domain certificate of Pix participants.`